integration-templates/github/github-issues-demo-action.ts

```diff
@@ -1,6 +1,6 @@
-import type { NangoSync, GithubWriteIssueDemoInput } from './models';
+import type { NangoSync, GithubWriteIssueInput, GithubWriteIssueResult } from './models';
 
-export default async function runAction(nango: NangoSync, input: GithubWriteIssueDemoInput): Promise<{ status: number }> {
+export default async function runAction(nango: NangoSync, input: GithubWriteIssueInput): Promise<GithubWriteIssueResult> {
```

Comment:
```
This Issue was created automatically using Nango's Action system.
```

Why is the "I" in issue capitalized. "Nango's Action system" -- never heard that phrasing before and reads a bit oddly. Can this reflect better what we call it in the docs?

Reply:
Changed the wording 👌🏻